odl/util/vectorization.py

```diff
@@ -228,6 +228,10 @@ class vectorize(OptionalArgDecorator):
     @staticmethod
     def _wrapper(func, *vect_args, **vect_kwargs):
         """Return the vectorized wrapper function."""
+        if not hasattr(func, '__name__'):
+            # Set name if not available. Happens if func is actually a callable
+            func.__name__ = '{}.__call__'.format(func.__class__.__name__)
```

Comment:
So same question here: why append `.__call__` to the name? Technically, neither `Class.__call__` nor `Class` will be usable directly in the wrapper, hence appending `__call__` does not really make the printed construction work literally.